src/cloud/index.js

```diff
@@ -270,3 +281,46 @@ export async function logs(options) {
   if (!confirmed) process.exit(0);
   await open(url);
 }
+
+export async function secret(options) {
+  await assertBedrockRoot();
+
+  const environment = options.environment || (await getEnvironmentPrompt());
```

Comment:
shouldn't need parens around await here I think

Reply:
You are right, and that is what I did until my editor (prettier) slapped parentheses around it.

Reply:
https://github.com/prettier/prettier/issues/3089

guess this is a known issue... one more reason I'm not the biggest fan of prettier :)